src/nvim/eval.c

```diff
@@ -17037,18 +17037,6 @@ long get_vim_var_nr(int idx) FUNC_ATTR_PURE
   return vimvars[idx].vv_nr;
 }
 
-/// Get typval_T representing v: variable
-///
-/// @warning if v: variable has reference counter it is not increased.
-///
-/// @param[in]  idx  Variable index, @see VimVarIndex.
-typval_T get_vim_var_tv(VimVarIndex idx) FUNC_ATTR_PURE
```

Comment:
@ZyX-I Why remove this? The note about reference counter was a useful hint, which is more likely to be missed now. And this function reduced verbosity at the call site.